src/tool_operate.c

```diff
@@ -1260,6 +1267,9 @@ static CURLcode single_transfer(struct GlobalConfig *global,
           break;
 
         use_proto = url_proto(per->this_url);
+        result = use_proto.result;
+        if(result)
+          break;
```

Comment:
This is new behavior. We do not error on this in existing code because it allows libcurl to have the final say if the URL works or not.

Reply:
Should i remove this check in this commit?
In https://github.com/curl/curl/pull/8805 i would then check for the protocol first (same style like `if(use_proto.proto == proto_scp || use_proto.proto == proto_sftp)`) and only in that case handle the return type?

In other terms: Handle return type on IPFS specific case, ignore otherwise?